classes/submission/reviewAssignment/ReviewAssignment.php

```diff
@@ -233,23 +230,21 @@ public function setReviewRoundId($reviewRoundId)
     }
 
     /**
-     * Get reviewer recommendation.
+     * Get reviewer recommendation id
      *
      * @return string
```

Comment:
The `@return` is incorrect after this change. In this case it doesn't add anything, now that there's a typehint, so it should just be removed.